### ISSUE TITLE
werf deploy is finished before the resources start

### ISSUE THREAD

**Issue by sasokol** [werf/werf#3506]

**Comment by distorhead** [comment]
Some info:

```
spec:
  progressDeadlineSeconds: 2147483647
  replicas: 4
  revisionHistoryLimit: 2
  selector:
    matchLabels:
      app: myapp
  strategy:
    rollingUpdate:
      maxSurge: 2
      maxUnavailable: 1
    type: RollingUpdate
```

Need a way to reproduce.

**Comment by distorhead** [comment]
Fixed in v1.1 and v1.2 by following PRs:
https://github.com/werf/werf/pull/3583
https://github.com/werf/werf/pull/3582
https://github.com/werf/kubedog/pull/209
https://github.com/werf/kubedog/pull/208
